tests/test_scribeauth.py

```diff
@@ -1,23 +1,29 @@
 import os
 import unittest
+from time import sleep
 
+import pyotp
 from botocore.awsrequest import AWSRequest
 from dotenv import load_dotenv
+
 from scribeauth import (
+    MissingIdException,
+    ResourceNotFoundException,
     ScribeAuth,
     Tokens,
```

Comment:
_Tokens_ is imported but we're not using this line anymore in the tests

Reply:
they probably will when you fix the types